airflow/providers/amazon/aws/example_dags/example_ecs_fargate.py

```diff
@@ -14,61 +14,39 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-"""
-This is an example dag for EcsOperator.
-
-The task "hello_world" runs `hello-world` task in `c` cluster.
-It overrides the command in the `hello-world-container` container.
-"""
```

Comment:
Is there a reason to remove it?
Since we have `dag.doc_md = __doc__` this explanatory block will appear in the UI

Reply:
It's the only sample DAG in the AWS provider package that had it, and I didn't see any added value in this instance.

Reply:
Discussed it a little on Slack here:  https://apache-airflow.slack.com/archives/CJ1LVREHX/p1644527546510599